### ISSUE TITLE
Update the "How does it work" section 

### ISSUE THREAD

**Issue by sparkO7** [DBenMoshe/contrast-ratio-repo#117]
The following text is missing from the How-does-it-work section: 
```
The Contrast Ratio Calculator will calculate the contrast, ranging from 1 (no contrast) to 21 (black on white).
```

Please, add a new `p` element with the text above, just after the list with the color formats.

**Comment by thetalesofj** [comment]
Do you want know of the text highlighted or in bold?

**Comment by thetalesofj** [comment]
Do you want any of the text in backticks or bold?
